src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -72,6 +73,14 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
 
   const overrideClassName = backwardCompatibilityForProperties([className, componentClassName]);
 
+  const overrideIcon = useMemo(() => {
+    if (type === AttentionBox.types.PRIMARY) {
+      return InfoIcon;
+    }
+
+    return icon;
+  }, [type, icon]);
```

Comment:
```suggestion
  const defaultIcon = useMemo(() => {
    return type === AttentionBox.types.PRIMARY ? InfoIcon : AlertIcon;
  }, [type]);
  const overrideIcon = icon ?? defaultIcon;
```

Cause currently you are overriding any icon if `type === AttentionBox.types.PRIMARY` and you need to only provide a default value for this case